Text for security considerations (#139)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

# Security Considerations

An improperly configured or malicious data sender could cause a

> data receiver to  acknowledge more frequently than its available resources

Reviewer: ```suggestion
data receiver to acknowledge more frequently than its available resources
```

permits. However, there are two limits that make such an attack largely
inconsequential. First, the acknowledgement rate is bounded by the rate at which

> data is received. Second, an ACK_FREQUENCY frame can only request an increase in
> the acknowledgment rate but cannot force it.

Reviewer: ```suggestion
data is received. Second, ACK_FREQUENCY and IMMEDIATE_ACK frames can only request an increase in
the acknowledgment rate, but cannot force it.
```

# Security Considerations

An improperly configured or malicious sender could cause standards-compliant

> receivers to  acknowledge more frequently than their available resources permit,
> thus causing a denial-of-service (DoS) attack. However, a sender has control of

Reviewer: ```suggestion
receivers to consume all available resources sending acknowledgements,
thus causing a denial-of-service (DoS) attack. However, a sender has control of
```

many aspects of the receiver behaviour and therefore, for a well-configured
sender, this introduces no additional threat. Further, the information in the
ACK_FREQUENCY frame are ultimately only a request but do not provide any ability

> for the requester to enforce a certain behavior on the ACK sender. As such an

Reviewer 2: It's a MUST earlier in the text though:
> On receiving an ACK_FREQUENCY frame and updating its recorded max_ack_delay and Ack-Eliciting Threshold values ({{multiple-frames}}), the endpoint MUST send an acknowledgement when one of the following conditions are met:
> 
> * Since the last acknowledgement was sent, the number of received ack-eliciting packets is greater than or equal to the recorded Ack-Eliciting Threshold.

Do we need to weaken that language a bit to allow the receiver more flexibility?

Author: I was wondering about this as well. I will open a separate issue!

Reviewer: Issue #144
